js/home.js

```diff
@@ -0,0 +1,39 @@
+var projectArtciles= [];
+
+function Project (projectInfo) {
+  this.title= projectInfo.title;
+  this.image= projectInfo.image;
+  this.link= projectInfo.link;
+  this.date= projectInfo.date;
+  this.description= projectInfo.description;
+  this.technologies= projectInfo.technologies;
+  this.repo= projectInfo.repo;
+};
+
+function handleMainNav() {
```

Comment:
move this to a separate JS file and attache to an object, like `articleView.js` in the labs